java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -690,6 +690,286 @@ void testInnerJoinWithOnlyCommonKeys() {
     }
   }
 
+  @Test
+  void testLowerBound() {
+    boolean[] descFlags = new boolean[3];
```

Comment:
could each `try` be a separate test method?

Reply:
I thought about that and either works. Will update the PR.

Reply:
Ideally this could be a template.

Reply:
@abellina  I have done an in-between solution to what u suggested so that we can still tie the different flavors for bounds under a single test while keeping them reusable for lower and upper bound and removed redundancy. Let me know if that works.

Reply:
Made them separate tests @abellina